matter/efr32/efr32mg24/BRD4187C/autogen/sl_component_catalog.h

```diff
@@ -21,16 +22,20 @@
 #define SL_CATALOG_BLUETOOTH_FEATURE_SYSTEM_PRESENT
 #define SL_CATALOG_BLUETOOTH_PRESENT
 #define SL_CATALOG_BLUETOOTH_FEATURE_MULTIPROTOCOL_PRESENT
+#define SL_CATALOG_GECKO_BOOTLOADER_INTERFACE_PRESENT
 #define SL_CATALOG_DEVICE_INIT_NVIC_PRESENT
 #define SL_CATALOG_EMLIB_CORE_DEBUG_CONFIG_PRESENT
 #define SL_CATALOG_FREERTOS_KERNEL_PRESENT
 #define SL_CATALOG_KERNEL_PRESENT
 #define SL_CATALOG_GATT_CONFIGURATION_PRESENT
-
 #ifdef DISPLAY_ENABLED
 #define SL_CATALOG_MEMLCD_USART_PRESENT
+#define SL_CATALOG_DMD_MEMLCD_PRESENT
 #endif
+#define SL_CATALOG_NVM3_PRESENT
+#define SL_CATALOG_PSA_CRYPTO_PRESENT
 #define SL_CATALOG_RAIL_LIB_PRESENT
+#define SL_CATALOG_RAIL_UTIL_PTI_PRESENT
```

Comment:
#define SL_CATALOG_SE_MANAGER_PRESENT is still there

Reply:
done